### PR TITLE
Possibility to build other content

### PR DESCRIPTION
**Pull request by tadatuta**
Depends on
* https://github.com/bem-site/gorshochek/pull/61
* https://github.com/bem-site/gorshochek/pull/62

Implementation is far from good but kinda works for urgent task of @levonet.

// cc @vithar

### CODE REVIEW

File: lib/data-builder.js [hunk 1]
```diff
@@ -17,7 +17,10 @@ const params = process.env;
 Q()
     .then(tasks.core.mergeModels(model, {modelPath: params.modelPath}))
     // .then(tasks.meta.generateTagPages(model)) // Включить когда в модели появятся теги
-    .then(tasks.docs.loadSourceFromGithub(model, {token: params.token}))
+    .then(tasks.docs.loadSourceFromGithub(model, {
+        token: params.token,
+        githubHosts: [{ url: 'github.yandex-team.ru', host: 'api.github.yandex-team.ru' }]
```
Comment (vithar) [inline lib/data-builder.js]: !

File: lib/prepare-model.js [hunk 2]
```diff
@@ -30,6 +30,11 @@ module.exports = function(model, lang) {
                 return;
             }
 
+            if (typeof value === 'string') {
```
Comment (vithar) [inline lib/prepare-model.js]: Add comment

**Comment by tadatuta** [comment]
⬆️

**Comment by tadatuta** [comment]
⬆️ 🆙 ⏫

**Pull request merged**